tests/Medidata.ZipkinTracer.Tests/Handlers/HttpContextInjectorMiddleware.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web;
+using Medidata.ZipkinTracer.Core;
+using Medidata.ZipkinTracer.Owin;
+using Microsoft.Owin;
+using Owin;
+using Rhino.Mocks;
```

Comment:
Not sure whether this is out of scope but, I think we need to move away from using Rhino Mocks whenever possible since it is no longer actively developed since 2011 according to author himself.
[The state of Rhino Mocks](https://ayende.com/blog/161826/the-state-of-rhino-mocks)